### DISCUSSION TITLE
怎么还用.sync修饰符呢 应该改成v-model吧

### DISCUSSION THREAD

**Discussion by lovetingyuan** [element-plus/element-plus#1552] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7310471/109637768-3337dd80-7b88-11eb-8299-f53b6a8134cf.png)
vue3不支持sync修饰符的啊，这是文档错误还是实现有问题，应该用v-model才对

https://element-plus.gitee.io/#/zh-CN/component/pagination

**Comment by jw-foss** [comment, marked as answer]
Right, that was a mistake.
I think you should raise an issue instead of a discussion to talk about this.
Anyways, you interested in fixing this doc?

**Reply by zhazhanitian** [reply]
I want to be a contributor, but I don't know where to start. Can I take some tasks from you as a breakthrough. For example, assign some issues for me to complete

**Reply by zhazhanitian** [reply]
For example, can I propose a PR to fix the above problem

**Reply by iamkun** [reply]
@zhazhanitian
Thanks, I've added you to this issue to track this on.

https://github.com/element-plus/element-plus/issues/1920
